Libplanet.Extensions.Cocona/Commands/KeyCommand.cs

```diff
@@ -66,23 +98,50 @@ public KeyCommand()
             }
         }
 
-        [Command(Description = "Import a raw private key.")]
+        [Command(Description = "Import a raw private key or Web3 Secret Storage.")]
         public void Import(
             [Argument(
                 "PRIVATE-KEY",
-                Description = "A raw private key to import in hexadecimal string."
+                Description = @"A raw private key in hexadecimal string, or path to Web3 Secret
+                Storage to import"
```

Comment:
In C# language, the `@` keyword makes us able to use all strings until the end character `"` as raw. For examples:

```csharp
Console.WriteLine(@"Hello
World");
```

The above code will print:

```
Hello
World
```

And in the indented case, it will print all of them without modification.

```csharp
Console.WriteLine(@"Hello
       World");
```

```
Hello
       World
```

So I saw the help message like the below text and I thought it should be checked once.

```
Usage: dotnet key import [--passphrase <PASSPHRASE>] [--passphrase-file <FILE>] [--json] [--dry-run] [--path <String>] [--help] private-key

Import a raw private key or Web3 Secret Storage.

Arguments:
  0: private-key    A raw private key in hexadecimal string, or path to Web3 Secret
                Storage to import (Required)

Options:
  -p, --passphrase <PASSPHRASE>    Take passphrase through this option instead of prompt.  Mutually exclusive with --passphrase-file option.
  --passphrase-file <FILE>         Read passphrase from the specified file instead of taking it through prompt.  Mutually exclusive with -p/--passphrase option.  For standard input, use a hyphen (`-').  For an actual file named a hyphen, prepend `./', i.e., `./-'.  Note that the trailing CR/LF is trimmed.
  --json                           Import Web3 Secret Storage key.
  --dry-run                        Do not add to the key store, but only show the created key.
  --path <String>                  Path to key store
  -h, --help                       Show help message
```

If you didn't want the newline and indents and you used it to avoid line length limitation, how about using `+` operator to split string? like...
```suggestion
                Description = "A raw private key in hexadecimal string, or path to Web3 Secret " +
                              "Storage to import"
```